src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt

```diff
@@ -0,0 +1,738 @@
+package me.zeroeightsix.kami.module.modules.misc
+
+import baritone.api.BaritoneAPI
+import me.zeroeightsix.kami.KamiMod
+import me.zeroeightsix.kami.event.events.RenderEvent
+import me.zeroeightsix.kami.module.Module
+import me.zeroeightsix.kami.module.ModuleManager
+import me.zeroeightsix.kami.module.modules.player.InventoryManager
+import me.zeroeightsix.kami.module.modules.player.LagNotifier
+import me.zeroeightsix.kami.module.modules.player.NoBreakAnimation
+import me.zeroeightsix.kami.setting.Settings
+import me.zeroeightsix.kami.util.BlockUtils
+import me.zeroeightsix.kami.util.InventoryUtils
+import me.zeroeightsix.kami.util.color.ColorHolder
+import me.zeroeightsix.kami.util.graphics.ESPRenderer
+import me.zeroeightsix.kami.util.graphics.GeometryMasks
+import me.zeroeightsix.kami.util.math.CoordinateConverter.asString
+import me.zeroeightsix.kami.util.math.RotationUtils
+import me.zeroeightsix.kami.util.text.MessageSendHelper
+import net.minecraft.block.Block
+import net.minecraft.block.Block.getBlockById
+import net.minecraft.block.Block.getIdFromBlock
+import net.minecraft.block.BlockAir
+import net.minecraft.block.BlockLiquid
+import net.minecraft.client.audio.PositionedSoundRecord
+import net.minecraft.entity.item.EntityItem
+import net.minecraft.entity.item.EntityXPOrb
+import net.minecraft.init.SoundEvents
+import net.minecraft.network.play.client.CPacketEntityAction
+import net.minecraft.network.play.client.CPacketPlayerDigging
+import net.minecraft.util.EnumFacing
+import net.minecraft.util.EnumHand
+import net.minecraft.util.ResourceLocation
+import net.minecraft.util.math.AxisAlignedBB
+import net.minecraft.util.math.BlockPos
+import net.minecraft.util.math.Vec3d
+import net.minecraftforge.fml.common.registry.ForgeRegistries
+import java.util.*
+import java.util.stream.IntStream.range
+import kotlin.math.floor
+import kotlin.math.roundToInt
+
+/**
+ * @author Avanatiker
+ * @since 20/08/2020
+ */
+
+@Module.Info(
+        name = "HighwayTools",
+        description = "Even Better High-ways for the greater good.",
+        category = Module.Category.MISC
+)
+class HighwayTools : Module() {
+
+    private val mode = register(Settings.e<Mode>("Mode", Mode.HIGHWAY))
+    private val page = register(Settings.e<Page>("Page", Page.MAIN))
+    //Settings for module
+    val baritoneMode = register(Settings.booleanBuilder("Baritone").withValue(true).withVisibility { page.value == Page.MAIN }.build())
```

Comment:
this should be more descriptive. the user can't tell what the `Baritone` setting does from this